mmv1/templates/terraform/examples/dialogflowcx_version_regional.tf.erb

```diff
@@ -9,6 +9,11 @@ resource "google_dialogflow_cx_agent" "agent" {
   speech_to_text_settings {
     enable_speech_adaptation = true
   }
+  advanced_settings {
```

Comment:
Can we limit setting this new value to `_full` tests and/or dedicated tests?

Reply:
Unfortunately no:
1. this is a bit of weirdness in the API, b/303056144 if you want to look at the bug report. TLDR: sending a null loggingSettings sets `enableInteractionLogging = true` because reasons -> we need `advanced_settings { logging_settings {} }`.
2. there's a top-level `enable_stackdriver_logging = true` setting that will auto-set `advancedSettings.loggingSettings.enableStackdriverLogging = true` if you don't explicitly set it yourself -> we'll have a diff.

Hope that made sense.

Reply:
Ah- we'll need to make the new field `default_from_api` in that case, which will make removing the value possible. As-is this is a breaking change, since existing users' configurations would be forced to be updated following an update of the Terraform provider. (This would be fine if those users had explicitly enabled the setting out of band, say with gcloud, but any Terraform user could be affected as-is)